### ISSUE TITLE
troubles with imports or installation(angular 2.4.1)

### ISSUE THREAD

**Issue by vicabelotelova** [sweetalert2/ngx-sweetalert2#49]
i did this:
```bash
npm install --save sweetalert2 @toverux/ngx-sweetalert2
```
and this in my app.module:
```code
import { SweetAlert2Module } from '@toverux/ngx-sweetalert2';
imports: [
   ...........
    SweetAlert2Module.forRoot()
  ],
```

and i had this:
![image](https://user-images.githubusercontent.com/20780164/35725422-6ad8f6d0-0812-11e8-9f6a-9fe076f879de.png)

**Comment by toverux** [comment]
Hello @vicabelotelova :)

Unfortunately, ngx-sweetalert2 is no longer compatible with Angular 2.x. You have to upgrade to Angular 4 or 5 to use the latest versions.

The latest version to be compatible with Angular 2 was v.1.2.2, see the documentation here : https://github.com/sweetalert2/ngx-sweetalert2/tree/v.1.2.2

The package name has changed too, so instead of `@toverux/ngx-sweetalert2` you'll have to install `@toverux/ngsweetalert2@~1.2.2`.
Other change, SweetAlert2 wasn't, at the time, a peer dependency of the library: the library had it in its dependencies, meaning that you'll have to uninstall ngx-sweetalert2 and sweetalert2 and just reinstall the correct version of ngsweetalert2.

Basically:
```sh
npm uninstall -S sweetalert2 @toverux/ngx-sweetalert2 && npm i -S @toverux/ngsweetalert2
```

An then please follow the old documentation, not the current one.

I hope I'm clear enough :slightly_smiling_face:  Yes, there were a lot of changes. The library has evolved a lot since version 1.2.2. If it is possible, please consider upgrading your Angular installation at some time, it will be a lot easier to use a lot of recent packages !
